thegreenweb/manifest.json

```diff
@@ -42,15 +42,16 @@
     "images/top-logo-greenweb.png"
   ],
   "permissions": [
-        "https://api.thegreenwebfoundation.org",
-        "https://www.thegreenwebfoundation.org",
+        "*://api.thegreenwebfoundation.org/*",
+        "*://www.thegreenwebfoundation.org/*",
         "webNavigation",
         "tabs",
         "storage"
   ],
   "content_scripts": [
    {
       "matches": [
+            "*://*.duckduckgo.com/*",
```

Comment:
@nieebel is duckduckgo using the google engine? As this adds the google.js file. 
Did you test this? 
I haven't checked it myself, but cool if this works.

Reply:
whoops, this should not go to master. I just did some small test, but it doesn't work. 
I have removed it again: https://github.com/thegreenwebfoundation/web-extension/pull/37/commits/03549378b2cdaf7d6e8a11a80b5560f8500d1799